solarforecastarbiter/io/fetch/nwp.py

```diff
@@ -591,19 +659,40 @@ def _optimize_netcdf(nctmpfile, out_path):
         else:
             logger.info('Moving on to next model run')
             inittime = await next_run_time(inittime, modelpath, model)
+
+
+async def run(basepath, model_name, chunksize, once=False):
+    session = make_session()
+    modelpath = basepath / model_name
+    if model_name != 'gefs':
```

Comment:
Ok with me to conditional on 'gefs'. Another idea is to use the existence or truthiness of `members` to determine the control flow.